model_demos/cv_demos/monodle/pytorch_monodle.py

```diff
@@ -0,0 +1,58 @@
+import pybuda
+from pybuda.verify.backend import verify_module
+from pybuda import VerifyConfig
+from pybuda.verify import TestKind
+from pybuda._C.backend_api import BackendDevice
+import os
+import requests
+import torchvision.transforms as transforms
+from PIL import Image
+
+from cv_demos.monodle.utils.model import CenterNet3D
+
+
+def run_monodle_pytorch():
+    model_name = "monodle_pytorch"
+
+    # Set PyBuda configuration parameters
+    compiler_cfg = pybuda.config._get_global_compiler_config()
+    compiler_cfg.balancer_policy = "Ribbon"
+    compiler_cfg.enable_auto_fusing = False
+    compiler_cfg.default_df_override = pybuda._C.Float16_b
+    os.environ["PYBUDA_RIBBON2"] = "1"
+
+    available_devices = pybuda.detect_available_devices()
+    if available_devices:
+        arch = available_devices[0]
+        if arch == BackendDevice.Wormhole_B0:
+            os.environ["PYBUDA_DISABLE_CONSTANT_FOLDING"] = "1"
+            os.environ["TT_BACKEND_OVERLAY_MAX_EXTRA_BLOB_SIZE"] = f"{88*1024}"
+        elif arch == BackendDevice.Grayskull:
+            raise NotImplementedError("Model not supported on Grayskull")
+
+    # Load data sample
+    url = "https://images.rawpixel.com/image_1300/cHJpdmF0ZS9sci9pbWFnZXMvd2Vic2l0ZS8yMDIyLTA1L3BkMTA2LTA0Ny1jaGltXzEuanBn.jpg"
+    image = Image.open(requests.get(url, stream=True).raw)
+
+    # Preprocessing
+    transform = transforms.Compose(
+        [
+            transforms.Resize(256),
+            transforms.CenterCrop(224),
+            transforms.ToTensor(),
+            transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+        ]
+    )
+    img_tensor = transform(image).unsqueeze(0)
+
+    # Load model and prepare for evaluation (inference)
+    pytorch_model = CenterNet3D(backbone="dla34")
+    pytorch_model.eval()
+
+    # Create pybuda.PyTorchModule using the loaded Pytorch model
+    tt_model = pybuda.PyTorchModule(model_name, pytorch_model)
+
+    # run inference on Tenstorrent device
+    output_q = pybuda.run_inference(tt_model, inputs=[(img_tensor,)])
+    output = output_q.get()[0].value()
+    print(output)
```

Comment:
@milank94 can you provide some feedback on structuring this demo? :))

Reply:
Looks like the output here will be the same at this: https://github.com/xingyizhou/CenterNet?tab=readme-ov-file#use-centernet

Can we confirm if this is the output: `ret will be a python dict: {category_id : [[x1, y1, x2, y2, score], ...], }`?

I'm thinking that the post-processing is coming from here: https://github.com/xingyizhou/CenterNet/blob/master/src/tools/vis_pred.py

If we can confirm that output schema is `{category_id : [[x1, y1, x2, y2, score], ...], }` or something like that, then we can just leave it as is. Or we can use that post-processing to generate the output image with annotations.

Reply:
@ashokkumarkannan1 please provide more details when you find time :))

Reply:
Sure @nvukobratTT  . I will take a look into it today.

Reply:
Hi @milank94 , @nvukobratTT ,
I have researched the [CenterNet](https://github.com/xingyizhou/CenterNet?tab=readme-ov-file#use-centernet) for the postprocessing. But it seems [heads](https://github.com/xingyizhou/CenterNet/blob/4c50fd3a46bdf63dbf2082c5cbb3458d39579e6c/src/lib/opts.py#L302) used in CenterNet and Monodle seems different. So we can't use this post processing in Monodle model.

Reply:
Thanks for the feedback @ashokkumarkannan1! 

@milank94 I'll merge these changes into the tracking branch. We need it merged in order to keep track of this model support on our nightly runs. We can discuss over the details before we release this model. I.e. do we need to add more modifications or not